recipes/cdat_info/build.sh

```diff
@@ -0,0 +1,214 @@
+export CFLAGS="-Wall -g -m64 -pipe -O2  -fPIC"
+export CXXLAGS="${CFLAGS}"
+export CPPFLAGS="-I${PREFIX}/include"
+export LDFLAGS="-L${PREFIX}/lib"
+
+export GIT_DESCRIBE=`git describe --tags`
+
+cat > cdat_info.py.in << EOF
```

Comment:
> . But cdat_info is sort of a reference for all pacakges to know what's going on. for example they all use pingDB/get_sample_data_path, etc.. function to send (if user approved of course) anonymous usage info to help us improve the software.

I guess that, as a packager, I don't really get what you are doing with that script. It does seems like it should be its own standalone package though.

> I'm not sure what your issue against the package is?

I don't understand the question. My comment is that `cdat_info.py.in` contains a lot of instructions to be in the `build.sh`. If they make sense as a standalone package or as part of the upstream software it would make more sense. If not, then we should add it as a separate file here for maintainability's sake.